plugins/aggregators/histogram/histogram_test.go

```diff
@@ -83,7 +84,7 @@ func TestHistogram(t *testing.T) {
 	histogram.Push(acc)
 
 	if len(acc.Metrics) != 6 {
-		assert.Fail(t, "Incorrect number of metrics")
+		require.Fail(t, "Incorrect number of metrics")
 	}
```

Comment:
Can we please convert this whole part to
```golang
	require.Len(t, acc.Metrics, 6, "Incorrect number of metrics")
```
same for the other occurrences below.

Reply:
Done.